docs/book/src/reference/creating-events

```diff
@@ -0,0 +1,109 @@
+# Creating Events
+
```

Comment:
Thank you for your contribution 🥇 .

Following are some comments:

- 1) You need to be able to verify and validate your changes. See that when you push a PR you can visualize how the docs will be within: (Click on Details)

<img width="912" alt="Screenshot 2023-05-01 at 08 59 31" src="https://user-images.githubusercontent.com/7708031/235426238-fd93af9d-4367-4c41-9a0a-912ef636755d.png">

- 2 the doc is missing an entry in the left menu. Can you please add?

Reply:
i dont know how to enter this in left menu. I tried comparing wit quick start for refernece. still dont understand. kindly explain.

Reply:
First click on in the details of the CI job for you are able to check the preview of the docs within the changes. Then, look for any left menu title and research for that in the code source so that you will find the file and be able to see how it works.

Reply:
I am working on this past one hour and still didnt get any left navigation syntax which needed to be added. I tried searching in other pages of reference like generating crds etc. They all just have only one syntax and that is, <aside = note>. I tried searching it on internet and found the 
```
---
tittle: creating-events.md
---
```
method as well. but its not used in other pages. Let me know what I missed now.

Reply:
Following the steps:

#### How to discover how the menu is built

- 1 - Go in the link: https://book.kubebuilder.io/
- 2 - Check any title in the left menu: 

<img width="338" alt="Screenshot 2023-05-04 at 07 28 12" src="https://user-images.githubusercontent.com/7708031/236126986-9c5e7d32-59dd-4688-a8f2-0fbc99b44aa5.png">

- 3 - Now use the search in your IDE or the repo into github: https://github.com/search?q=repo%3Akubernetes-sigs%2Fkubebuilder%20Available%20Plugins&type=code

- 4 - EURAKA You find all files where you have the same text. So you can discover how, where the menu is done: 

So, the menu is built in the SUMMARY.md file: https://github.com/kubernetes-sigs/kubebuilder/blob/06f81644f0fc2ea8fd07b94c4362535a181ec4f7/docs/book/src/SUMMARY.md?plain=1#L114-L120

Therefore, you need to add a NEW entry for a NEW section, Events. Where? Under references.

#### How to know how to write documentation

- 1 - Now check the docs: https://book.kubebuilder.io/
- 2 - See also the migration docs: https://book.kubebuilder.io/migration/manually_migration_guide_gov3_to_gov4.html
- 3 - See how the steps are provided. Please look at the style used and see how the info is described. 


IMPORTANT: Ensure that you read out the CONTRIBUTING guide: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#how-to-contribute-to-docs and check out:

> There are certain writing style guidelines for Kubernetes documentation, check out [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) for more information.  


#### Check out your changes. You must validate if your changes are OK (review it)


- 1 - You need to be able to verify and validate your changes. See that when you push a PR, you can visualize how the docs will be within: (Click on Details)

<img width="912" alt="Screenshot 2023-05-01 at 08 59 31" src="https://user-images.githubusercontent.com/7708031/235426238-fd93af9d-4367-4c41-9a0a-912ef636755d.png">

- 2 - Check out and do your review to see if all IS OK. Then, if so, you ask for reviews. Before that, you can update the title of your PR within (WIP) to make clear that you are still working on it.